common/traits/01_ancient_magic_traits.txt

```diff
@@ -227,39 +231,6 @@ mana_devouring_2 = {
 	level = 2
 }
 
-mana_burn = {
```

Comment:
trait that is prepared for future use, it will be caused if one uses spell over ones body capacity, in another sense if you cast spell which u cant cast without equipment or bilding it will cause mana burn